src/features/auth/ui/ResetPasswordForm.tsx

```
@@ -1,78 +1,102 @@
+      </div>
+      <div className="w-full flex items-center justify-start p-4 lg:p-8">
+        <div className="w-full max-w-[600px]">
+          <div className="border border-gray-700 rounded-[10px] shadow-2xl lg:bg-gray-600/30 lg:backdrop-blur-sm">
+            <div className="p-6">
+              <div className="space-y-5 min-h-[150px]">
+                <div>
+                  <label className="block text-sm font-medium text-gray-300 mb-2">
+                    새로운 비밀번호
+                  </label>
+                  <div className="relative">
+                    <Lock
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)